custom-server/llama3_eval.py

```diff
@@ -0,0 +1,312 @@
+# MODEL_ID= OPENAI_API_KEY= python3 llama3_eval.py --task gsm8k
```

Comment:
Although we already have Baseten Eval internally, this is a standalone file with no extra dependencies, making it more convenient for external users to use.